frontend/src/components/datatable/DeleteColumn.tsx

```diff
@@ -21,7 +22,7 @@ export default function DeleteColumn<T>(props: DeleteColumnProps<T>) {
   return (
     <Column
       body={rowDeleteButton}
-      header={"Delete"}
+      header={props.hideHeader ? "" : "Delete"}
```

Comment:
Why are we hiding header like this?

Reply:
I was experimenting with saving space, but I don't think it does much so I will remove it